25-hosted-ce-setup.sh

```diff
@@ -31,7 +31,7 @@ chmod 644 $CONDOR_SUDO_FILE
 
 echo "Running OSG configure.."
 # Run the OSG Configure script to set up bosco
-osg-configure -c
+osg-configure -c --verbose VERBOSE
```

Comment:
:facepalm: 
somebody forgot `action='store_true'` in osg-configure.

Reply:
lol, yeah I thought this was a weird way to invoke it